packages/app-root/src/app/users/[login]/stats/UserStatsContainer.js

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import { UserStats } from '@zooniverse/user'
+import { useContext } from 'react'
+
+import { PanoptesAuthContext } from '../../../../contexts'
+
+function UserStatsContainer({
+  login
```

Comment:
Not sure if we want to add `prop-types` to app-root?

Reply:
I'm okay either way! Is this prop required in UserStats?

Reply:
It is for admin access to a different user's stats page.